spring/src/test/java/io/kubernetes/client/spring/extended/controller/KubernetesReconcilerCreatorTest.java

```diff
@@ -12,24 +12,39 @@
 */
 package io.kubernetes.client.spring.extended.controller;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
```

Comment:
Can you remove this wildcard import in favor of the specific functions?

Reply:
updated